### ISSUE TITLE
A mistake?

### ISSUE THREAD

**Issue by AntonKanevets** [php/doc-ru#468]
From manual page: https://php.net/language.oop5.paamayim-nekudotayim

From manual:
The Scope Resolution Operator (also called Paamayim Nekudotayim) or in simpler terms, the double colon, is a token that allows access to [static](https://www.php.net/manual/en/language.oop5.static.php), [constant](https://www.php.net/manual/en/language.oop5.constants.php), and overridden properties or methods of a class.

I think it should be like this: 
The Scope Resolution Operator (also called Paamayim Nekudotayim) or in simpler terms, the double colon, is a token that allows access to [static](https://www.php.net/manual/en/language.oop5.static.php), [constant](https://www.php.net/manual/en/language.oop5.constants.php), and overridden properties or methods of a **_parent_** class.

**Comment by saundefined** [comment]
Hi,

Thanks for the question, this should be addressed to [doc-en](https://github.com/php/doc-en/issues/).
